BrowserKit/Sources/ContentBlockingGenerator/JsonHelper.swift

```diff
@@ -10,7 +11,10 @@ struct JsonHelper {
 
         do {
             let data = try Data(contentsOf: file)
-            return try JSONSerialization.jsonObject(with: data, options: []) as! [String: Any]
+            guard let jsonObject = try JSONSerialization.jsonObject(with: data, options: []) as? [String: Any] else {
+                return [:]
```

Comment:
i'd add a fatalError here so in case some if there is some file corruption we notice it from the crash